libdnf/system/state.hpp

```diff
@@ -190,6 +190,10 @@ class State {
     /// @since 5.0
     void save();
 
+    // TODO(mblaha) - needed to override modules state from dnf4 persistor
+    // remove the method from API once not needed
+    void reset_module_states(std::map<std::string, ModuleState> new_states) { module_states = new_states; }
```

Comment:
Please could you add a comment what it does or when it is supposed to be used?

Reply:
OK, I've documented the method and moved it into private.